projects/fourier/lib/fourier/services/lint/lockfiles.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+require "json"
+
+module Fourier
+  module Services
+    module Lint
+      class Lockfiles < Base
+        attr_reader :root_directory
+
+        def initialize(root_directory: Constants::ROOT_DIRECTORY)
+          @root_directory = root_directory
+        end
+
+        def call
+          spm_lockfile = self.spm_lockfile
+          tuist_lockfile = self.tuist_lockfile
+          assert_same_versions(spm_lockfile: spm_lockfile, tuist_lockfile: tuist_lockfile)
+        end
+
+        private
+          def assert_same_versions(spm_lockfile:, tuist_lockfile:)
+            mismatched_packages = []
+            common_packages = spm_lockfile.keys & tuist_lockfile.keys
```

Comment:
I think all the keys should be the same now, so I'd assert that the count of packages is the same as well